resources/istio/values.yaml

```diff
@@ -27,3 +27,11 @@ monitoring:
   istioServiceMonitor:
     enabled: true
     scrapeInterval: ""
+
+istio_kyma_patch:
```

Comment:
Since this is a test, maybe we should put it under `test.istio_kyma_patch` or better, rename it already to `istio_kyma_validate`